vyper/ast_utils.py

```diff
@@ -0,0 +1,141 @@
+import ast as python_ast
+
+import vyper.ast as vyper_ast
+from vyper.exceptions import (
+    ParserException,
+    SyntaxException,
+)
+from vyper.parser.parser_utils import (
+    annotate_and_optimize_ast,
+)
+from vyper.parser.pre_parser import (
+    pre_parse,
+)
+
+
+def parse_python_ast(source_code, node):
+    if isinstance(node, list):
+        o = []
+        for n in node:
+            o.append(
+                parse_python_ast(
+                    source_code=source_code,
+                    node=n,
+                )
+            )
+        return o
+    elif isinstance(node, python_ast.AST):
+        class_name = node.__class__.__name__
+        if hasattr(vyper_ast, class_name):
+            vyper_class = getattr(vyper_ast, class_name)
+            init_kwargs = {
+                'col_offset': getattr(node, 'col_offset', None),
```

Comment:
Is there a reason why these fields aren't reliably present on the `node` class?  That seems unfortunate.

Reply:
Main reason IIRC is that ast.Module doesn't have them. As there is no col_offset / line_no for that concept ;)

Reply:
```
>>> import ast
>>> a = ast.parse('a: int128 = 1')
>>> a.body
[<_ast.AnnAssign object at 0x7f55fb142748>]
>>> a.body[0]
<_ast.AnnAssign object at 0x7f55fb142748>
>>> a
<_ast.Module object at 0x7f55fb1426a0>
>>> a._
a._attributes  a._fields      
>>> a._attributes
()
>>> a._fields
('body',)
>>> a.body[0]._attributes
('lineno', 'col_offset')
```
Yep that's it